py_inspect.py

```diff
@@ -335,10 +360,34 @@ def __init__(self):
         self.action.addAction(self.mouse)
         self.action.addAction(default)
         self.action.addSeparator()
+
         self.bmethods = self.action.addMenu("Base Wrapper Methods")
-        self.hmethods = self.action.addMenu("Hwnd/Win32 Wrapper Methods")
+
+        self.hmethods = self.action.addMenu("Hwnd Wrapper Methods")
+        self.hmethods.menuAction().setVisible(False)
         self.umethods = self.action.addMenu("UIA Wrapper Methods")
+        self.umethods.menuAction().setVisible(False)
+        self.backend_menus = {
+            'last_used': self.umethods,
+            'win32': self.hmethods,
+            'uia': self.umethods,
+            # HOW TO ADD BACKEND
```

Comment:
Please mark such places with keyword "TODO". It will be easier to search for them.